atomata/capstone_WebClient
# Testsprint4 cont

### Summary 
2 main things. 
1. Created file overlayfunc which includes functions from the file overlay.tsx (refactor)
2. Code review, for the 2 functions since it needed to be annotations.
3. Create test for it (not completly doen yet)

### Details
The overlayfun.tsx contains 2 functions that were from the overlay.tsx including the needed hooks. 

### How to test?
Run the program and see if the program is running the way it should. 
To see the test. follow readme file
